tests/01_unit/13_outputs/test_tg_commands.py

```diff
@@ -18,20 +18,21 @@ def test_parse_quotted_message(quo):
     """Testing of parsing quotted message"""
     expected = ('awx', 'Scenario 1.5', 5)
     message = f'/status {expected[0]} {quo}{expected[1]}{quo} {expected[2]}'
-    source, detailed, count = parse_command(message)
-    assert (source, detailed, count) == expected
+    command = parse_command(message)
+    assert (command.target, command.detailed, command.depth) == expected
 
 
 @pytest.mark.parametrize(('message', 'expected'),
                          [
                              ('/status awx \'Scenario 1.5\' 5', ('awx', 'Scenario 1.5', 5)),
-                             ('/status awx', ('awx', None, 1)),
+                             ('/status awx', ('awx', '', 1)),
                              ('/status awx  "Destroy 1" ', ('awx', 'Destroy 1', 1))
                          ])
 def test_parse_command(message, expected):
     """Testing of parsing command arguments"""
-    source, detailed, count = parse_command(message)
-    assert (source, detailed, count) == expected
+    command = parse_command(message)
+    target, detailed, depth = expected
+    assert command == ParsedStatusCommand(target, detailed, depth)
```

Comment:
Better move it to parameters, like `('/status awx', ParsedStatusCommand('awx', '', 1))` and than chech it just with `assert command == expected`